src/util/file-util.ts

```diff
@@ -0,0 +1,19 @@
+import { readFileSync } from 'fs';
+import { S3 } from 'aws-sdk';
+
+export class FileUtil {
+
+    static async GetContents(filePath: string): Promise<string> {
+        if (filePath === undefined) {throw new Error('FileUtil.GetContents filePath is undefined');}
+
+        if (filePath.startsWith('s3://')) {
+            const s3client = new S3(); // todo: fix this
```

Comment:
could you provide more info on the `todo:`?  what's to be fixed?

It would be nice to be able to use a different profile in case the bucket is in a different AWS account.  However that can be done as an improvement in a follow on PR.

Reply:
Yeah, had not decided on this but thought the exact same. Maybe in a next iteration. M